packages/forms/src/UIForm/fields/Datalist/Datalist.component.js

```diff
@@ -74,25 +77,50 @@ class Datalist extends Component {
 	}
 
 	getTitleMap() {
-		const titleMap = this.state.titleMap || this.props.schema.titleMap || [];
+		const isMultiSection = this.props.schema.options && this.props.schema.options.multiSection;
+		const titleMap =
+			this.state.titleMap ||
+			((this.props.schema.options && this.props.schema.options.titleMap) ||
```

Comment:
same here i think using lodash get will be great for code readability and safety